code/modules/antagonists/heretic/status_effects/buffs.dm

```diff
@@ -292,3 +293,24 @@
 /datum/status_effect/caretaker_refuge/proc/prevent_cuff(datum/source, mob/attemptee)
 	SIGNAL_HANDLER
 	return COMSIG_CARBON_CUFF_PREVENT
+
+// Path Of Moon status effect which hides the identity of the heretic
+/datum/status_effect/moon_grasp_hide
+	id = "Moon Grasp Hide Identity"
+	status_type = STATUS_EFFECT_REFRESH
+	duration = 15 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/moon_grasp_hide
+
+/datum/status_effect/moon_grasp_hide/on_apply()
+	owner.add_traits(TRAIT_UNKNOWN, TRAIT_SILENT_FOOTSTEPS)
+	owner.update_name()
```

Comment:
AFAIK this update name call is completely superfluous (and also improper) so just, axe it
```suggestion
```

Reply:
Axed